src/ComposerRequireChecker/Cli/CheckCommand.php

```diff
@@ -68,6 +71,12 @@ protected function configure(): void
                 InputOption::VALUE_NONE,
                 'this will cause ComposerRequireChecker to ignore errors when files cannot be parsed, otherwise'
                 . ' errors will be thrown'
+            )
+            ->addOption(
+                'report-json',
+                null,
```

Comment:
Does `php://stdout` work, for writing to STDOUT for piping purposes? I can see how this feature is mostly useful when combined with `jq` :thinking:

Reply:
This would imply to suppress any other output if specified - or move “verbose” and other output to stderr. Otherwise no clean json output would be in stdout. I considered it but didn’t want to break BC here.

Reply:
Shouldn't break BC, since:

 1. it was not possible to pass in the argument before
 2. if you now pass it, it changes the output as expected

Therefore any behavioral change we couple to the new `--report-json` flag/parameter should be safe to introduce.

Reply:
I’m thinking about changing the option to `—output=json` and therefore only be able to have one output format at stdout. There would be “text” for existing behavior and “json” for new one (implying quiet output except for the report).

Reply:
Done. No more `--report-json=[file]`, only `--output={json|text}`.